### ISSUE TITLE
Homebridge a slow down / crashing

### ISSUE THREAD

**Issue by pcorrado83** [pdemarco925/homebridge-icomfort#15]
Been using this plug in for a couple of years without issue, however, about 5 months it started causing major slowdowns with my other accessories and also all out server crashes. Disabling the plug in resolve the issue - so it is related to this plug in. 

I know this plug in has not been updated for some time and that could be the issue but I figured I would see if anyone else had issues / a fix.

**Comment by DaMojoh** [comment]
Yes, I have the same exact experience with this app. It was great until recently and it is a shame, No updates or work-Arounds.

**Comment by cgjeffries** [comment]
Same here, worked fine for a few years, but suddenly this plugin is causing all accessories (even those from other plugins) to no longer respond in the Home app. Disabling it fixes the issue.
